submissions/benjaminKamena/database/init.sql

```diff
@@ -0,0 +1,132 @@
+--
+-- Database: `ecommerce_store`
+--
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `user_admins`
+--
+
+CREATE TABLE `user_admins` (
+  `admin_id` int(11) NOT NULL AUTO_INCREMENT,
+  `admin_name` varchar(100) NOT NULL,
+  `email_address` varchar(100) NOT NULL,
+  `admin_password` varchar(200) NOT NULL,
+  `phone_number` varchar(20) NOT NULL,
+  PRIMARY KEY (`admin_id`)
+);
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `products`
+--
+
+CREATE TABLE `products` (
+  `product_id` int(11) NOT NULL AUTO_INCREMENT,
+  `product_name` varchar(100) NOT NULL,
+  `product_desc` text NOT NULL,
+  `product_image` text NOT NULL,
+  `product_unit_price` decimal(6,2) NOT NULL,
+  `admin_id` int(11) NOT NULL,
+  `category_id` int(11) NOT NULL,
+  `stock_id` int(11) NOT NULL,
+  PRIMARY KEY (`product_id`),
+  FOREIGN KEY (`stock_id`) REFERENCES `stock_levels` (`stock_id`),
+  FOREIGN KEY (`category_id`) REFERENCES `categories` (`category_id`),
+  FOREIGN KEY (`admin_id`) REFERENCES `user_admins` (`admin_id`)
+);
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `stock_levels`
+--
+
+CREATE TABLE `stock_levels` (
```

Comment:
There is no need for this table. All the columns here can as well be on the products table